admin/config-ui/class-configuration-page.php

```diff
@@ -106,7 +106,7 @@ public function show_wizard() {
 		<html <?php language_attributes(); ?>>
 		<!--<![endif]-->
 		<head>
-			<meta name="viewport" content="width=device-width"/>
+			<meta name="viewport" content="width=device-width", initial-scale=1/>
```

Comment:
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`
The `initial-scale` needs to be within the double quotes.